README.rst

```diff
@@ -18,13 +18,35 @@ Disclaimer
 
 .. code::
 
-    $ mkdir -p ~/work/tahoe-hawthorn  # It needs its own new directory
-    $ cd ~/work/tahoe-hawthorn
+    $ mkdir -p ~/work/tahoe-juniper  # It needs its own new directory
+    $ cd ~/work/tahoe-juniper
     $ git clone https://github.com/appsembler/devstack.git
-    $ cd devstack  # Now the `devstack` repo should be on the `hawthorn` branch
+    $ cd devstack  # Now the `devstack` repo should be on the `juniper` branch
+    $ git checkout juniper
+
+Create an ``options.local.mk`` file in your local ``devstack`` directory:
+
+.. code::
+
+  OPENEDX_RELEASE=juniper.master
+  COMPOSE_PROJECT_NAME=junstack
+  DEFAULT_SERVICES=lms+studio+forum+amc+amc-frontend
+  DB_SERVICES=lms studio amc
+
```

Comment:
Luckily this is no longer _needed_ as of #58 so I'd rather remove it from here. You could keep your version of it, as I'm keeping mine. 

```suggestion
```